.gitignore

```diff
@@ -97,3 +96,7 @@ theOutput/
 
 # Ignore compilation stats from `yarn stats`
 compilation-stats.json
+
+# Ignore plugins sdk stuff
+./pkg/plugins/sdk/js/lib
+./pkg/plugins/sdk/node_modules
```

Comment:
Not sure where the `node_modules ` while actually be located now

Reply:
Isn't there still a node_modules directory produced in the root? If you keep the old entry in .gitignore, it should apply to all levels of the tree.

Reply:
Why are you changing the `node_modules` ignore pattern? The original pattern matches node_modules at the repo root for me, and it should also match at any other level of the tree. Unless I'm missing something, I think the original pattern should be kept.

Reply:
sorry, there was a conflict while a was merging it and I got confused

Reply:
@papagian OK :)

Reply:
Just fix it back then and I'll approve I guess.